### ISSUE TITLE
Using browser only shows error until end of the lesson

### ISSUE THREAD

**Issue by stazman** [learn-co-students/modification-nested-routes-reading-v-000#203]
Kept getting error of no "name" recognized in browser for posts#index post.author.name until HANDLING MISCHIEF AND ERRORS IN OUR URLS section was finished.

**Comment by catherinism** [comment]
I agree. I know this is a read_me but I still want to understand. I didn't see any error in the tests at all but just the browser. Still checking why I'm getting undefined method `posts_path'

**Comment by Terry-Thompson** [comment]
Students are unable to also follow along unless they add separate from the instruction, a resources post in config/routes.  Without it, you receive a NoMethodError, undefined method "posts_path".

**Comment by Sdcrouse** [comment]
Hi. Unless I'm mistaken, this seems to have been fixed; I got through the lab with virtually no problems.
